editor/find_in_files.cpp

```diff
@@ -874,6 +868,43 @@ void FindInFilesPanel::_on_replace_all_clicked() {
 	emit_signal(SNAME(SIGNAL_FILES_MODIFIED), modified_files);
 }
 
+void FindInFilesPanel::_on_button_clicked(TreeItem *p_item, int p_column, int p_id, int p_mouse_button_index) {
+	const String file_path = p_item->get_text(0);
+
+	if (_result_items.find(p_item)) {
+		_result_items.erase(p_item);
```

Comment:
You can avoid double lookup if you do
```suggestion
	HashMap<TreeItem *, Result>::Iterator E = _result_items.find(p_item);
	if (E) {
		_result_items.remove(E);
```